Detections/W3CIISLog/ProxyShellPwn2Own.yml

```diff
@@ -0,0 +1,42 @@
+id: 968358d6-6af8-49bb-aaa4-187b3067fb95
+name: Exchange ProxyShell Pwn2Own
+description: |
+  'This query looks for suspicious request patterns to Exchange servers that fit patterns recently
+  blogged about by PeterJson.
+  Reference: https://peterjson.medium.com/reproducing-the-proxyshell-pwn2own-exploit-49743a4ea9a1'
+severity: Medium
```

Comment:
Seems like this is High based on your comments in the PR submission?

Reply:
Increased to high severity, agreed.